project.inlang/project_id

```diff
@@ -0,0 +1 @@
+f03a621129f94818100684c3fabe02f1a98c13396942f4be7252902ffe86309e
```

Comment:
What is the use of this file?

Reply:
This file can be deleted safely. It was automatically added by inlang translator.